draft-ietf-gnap-core-protocol.md

```diff
@@ -204,29 +194,37 @@ role of the AS as defined by the protocol.
 In addition to the roles above, the protocol also involves several 
 elements that are acted upon by the roles throughout the process.
 
+Attribute
+: characteristics related to an end-user.
+
 Access Token
-: A credential representing a set of access rights
-delegated to the client instance. The access token is created by the AS, consumed
-and verified by the RS, and issued to and carried by the client instance. The contents
-and format of the access token are opaque to the client.
+: digitally signed data that contains specific rights and/or attributes.
+
+    Note: an access token can be first issued to an end-user (usually requiring his authentication) and subsequently refreshed.
+
+Claim
+: statement about a subject.
 
 Grant
-: The process by which the client instance requests and is given delegated
-access to the RS by the AS through the authority of the RO.
+: (verb): to permit an end-user to exercise some rights and/or to assert some attributes at a specific time and during a specific duration. (noun): the act of granting.
 
-Cryptographic Key
-: A cryptographic element binding a request to a
-holder of the key. Access tokens and client instances can be associated with
-specific keys.
+Privilege
+: right or attribute associated with an end-user.
```

Comment:
Are these associated with the end user or the resource owner?

Reply:
people seem to have different options about that. Right now I left it generic enough, but this could change in the future if we see we don't need to leave that open.

Reply:
To keep that generic, I suggest 
"right or attribute associated with a subject."

I also included a note
Note: the RO defines and maintains the rights and attributes associated to the protected resource, and might temporarily delegate some those privileges to an end-user. This process is refered to as privilege delegation. 

-> as a proposal to solve the delegated privilege issue